cmd/clusterctl/README.md

```diff
@@ -4,6 +4,19 @@
 
 Read the [experience doc here](https://docs.google.com/document/d/1-sYb3EdkRga49nULH1kSwuQFf1o6GvAw_POrsNo5d8c/edit#). To gain viewing permissions, please join either the [kubernetes-dev](https://groups.google.com/forum/#!forum/kubernetes-dev) or [kubernetes-sig-cluster-lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle) google group.
 
+## Goals
+
+* Provide a bootstrapping mechanism via `kind` or `minikube`
+* Provide an interface to install a CAPI provider into a cluster via a registry
+* Provide an interface to deploy and manage clusters from multiple providers
+
+## Non-Goals
+
+* Provide higher-order tools for multi-cluster workflows e.g.
+  * Web GUI
+  * User / RBAC Management
+  * Addon Management
```

Comment:
Addon Management is a tricky area, since there are certain "Addons" that we cannot avoid (such as CNI) which falls under the scope of `clusterctl` today.

Reply:
I think there are 2 camps:

1) `clusterctl` as a minimal cluster deployment tool that provides bootstrapping and pivot only.
2) `clusterctl` as a higher order manager of clusters that provides a good UX.

I am in favor of option 2, but either way, we should make a documented decision on it